tests/unit/match-test.js

```diff
@@ -23,17 +23,17 @@ test('should throw if the pattern does not match', function(assert) {
   const error = new Error('test');
 
   assert.throws(() => {
-    rethrow(error, { matcher: { message: 'bar' }, onMatch() {} });
+    match(error, { matcher: { message: 'bar' }, onMatch() {} });
   });
 });
 
-test('should not rethrow if the pattern matches when passed as an argument to catch', async function(assert) {
+test('should not match if the pattern matches when passed as an argument to catch', async function(assert) {
```

Comment:
This should still say `rethrow` IMO

Reply:
Ya, that's odd to say `match`. Will change.